lib/cli/commands/getnodeinfo.ts

```diff
@@ -1,6 +1,19 @@
 import { Arguments } from 'yargs';
 import { callback, loadXudClient } from '../command';
-import { GetNodeInfoRequest } from '../../proto/xudrpc_pb';
+import Table, { VerticalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { GetNodeInfoRequest, GetNodeInfoResponse } from '../../proto/xudrpc_pb';
+
+const displayNodeInfo = (node: GetNodeInfoResponse.AsObject) => {
+  const table = new Table() as VerticalTable;
+  const bannedTitle = colors.red('Banned');
+  const reputationScore = colors.red('Reputation Score');
+  table.push(
+    { [bannedTitle]: node.banned }
+  , { [reputationScore]: node.reputationscore });
+  console.log(colors.underline(colors.bold('\nNode info:')));
```

Comment:
I'm not sure whether this header line adds much value, but open to opinions. If we do keep it, maybe have it output the node pub key like `Node info for 02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6:`

![image](https://user-images.githubusercontent.com/3440835/58646895-60c96600-82d4-11e9-9904-bd9f1dcfc9b5.png)